docs/src/pages/en/styling.md

```diff
@@ -23,18 +23,19 @@ Our current styling architecture is still quite nascent and is very likely to un
-| `--media-button-icon-width`        | `width`       | none                                | default width of button icons                                                                     | Only applies to `<img>` and `<svg>` tags                                                                                                                             |
-| `--media-button-icon-height`       | `height`      | `var(--media-control-height, 24px)` | default height of button icons                                                                    | Only applies to `<img>` and `<svg>` tags                                                                                                                             |
-| `--media-icon-color`               | `fill`        | `#eee`                              | default fill color of button icons                                                                | Only applies to `<img>` and `<svg>` tags                                                                                                                             |
-| `--media-button-icon-transform`    | `transform`   | none                                | apply a [transform](https://developer.mozilla.org/en-US/docs/Web/CSS/transform) to button icons   | Only applies to `<img>` and `<svg>` tags                                                                                                                             |
-| `--media-button-icon-transition`   | `transform`   | none                                | apply a [transition](https://developer.mozilla.org/en-US/docs/Web/CSS/transition) to button icons | Only applies to `<img>` and `<svg>` tags                                                                                                                             |
+| Name                               | CSS Property  | Default Value                                      | Description                                                                                       | Notes                                                                                                                                                                |
+| ---------------------------------- | ------------- | -------------------------------------------------- | ------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `--media-control-background`       | `background`  | `var(--media-secondary-color, rgb(20 20 30 / .7))` | background color of the component                                                                 | Applies to other components as well ([See notes below \*\*](#notes))                                                                                                 |
```

Comment:
should `--media-primary-color` and `--media-secondary-color` get their own section at the top?

Reply:
Looks like the new font css vars aren't documented either. Thinking about this, might be fine to skip for now since we have an upcoming effort to update documentation for this.